playbooks/empty-dir-quota.yaml

```diff
@@ -0,0 +1,7 @@
+- hosts: schedulable_nodes
+  gather_facts: yes
+  become: yes
+  vars_files:
+    - vars/main.yaml
```

Comment:
Should be able to just remove them as it doesn't appear that the vars matter in regards to the play